From 168c40e50b6f97daebabfdb3276893c0e12977af Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Wed, 31 Mar 2010 13:42:50 +1000
Patch-mainline: 2.6.34
References: fate#310916
Git-commit: 08a370fa758fae7e387274ca3571ce625d7f829b
Subject: [PATCH 0554/2588] drm/radeon/kms: rs400/480 should set common
 registers.

These GPUs should be setting these registers up also.

Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit 08a370fa758fae7e387274ca3571ce625d7f829b)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/radeon/rs400.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/gpu/drm/radeon/rs400.c b/drivers/gpu/drm/radeon/rs400.c
index ff617b9..1a41cb2 100644
--- a/drivers/gpu/drm/radeon/rs400.c
+++ b/drivers/gpu/drm/radeon/rs400.c
@@ -391,6 +391,8 @@ static int rs400_startup(struct radeon_device *rdev)
 {
 	int r;
 
+	r100_set_common_regs(rdev);
+
 	rs400_mc_program(rdev);
 	/* Resume clock */
 	r300_clock_startup(rdev);
-- 
1.7.6

